Dockerfile

```
@@ -11,7 +11,8 @@ ENV CONTAINER_USER="analyticalplatform" \
     CONTAINER_GROUP="analyticalplatform" \
     CONTAINER_GID="1000" \
     DEBIAN_FRONTEND="noninteractive" \
-    MLFLOW_ROOT="/opt/mlflow"
+    MLFLOW_ROOT="/opt/mlflow" \
+    UV_VERSION="0.5.11"
```

```suggestion
    UV_VERSION="0.9.7"
```
https://github.com/astral-sh/uv/releases/tag/0.9.7